pkg/noun/vortex.c

```diff
@@ -120,6 +120,31 @@ _cv_nock_wish(u3_noun txt)
   return pro;
 }
 
+/* u3v_wish_n(): text expression with cache.
```

Comment:
update header comment to explain how the new fun differs from `u3v_wish`

Reply:
done